components/buttons/help.jsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable react/prop-types */
```

Comment:
There's an ESLint rule in the mix that requires that you define types (i.e. integer, string, boolean, etc) for props that are fed into your components. [You can read about the rule here!](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md)

I disabled these while building this out since the component tree and prop definitions were changing constantly. It's [on my todo list](https://github.com/kanadgupta/twin-cities-aid-distribution-locations/blob/8cb738911293f86aa7de13834a6da55e0551b033/issues.md#L33) to re-enable this rule and add prop types once we're good with this architecture!